python/cuml/cuml_accel_tests/upstream/umap/pytest.ini

```diff
@@ -0,0 +1,2 @@
+# Empty pytest config file so pytest doesn't use the pytest config for cuml itself
+[pytest]
```

Comment:
Without this our pytest config for `cuml` itself gets picked up. This causes some unnecessary failures due to warnings. IMO we should run the upstream test suite with the same configuration they use to test.

Reply:
It might be cleaner to force pytest to use a certain `--rootdir=` here. https://docs.pytest.org/en/stable/reference/customize.html#initialization-determining-rootdir-and-configfile

Reply:
Specifying the `rootdir` doesn't seem to stop pytest from continuing to walk up the directories until it finds a config file (this is also what the docs say). `umap` upstream doesn't do any pytest configuration, so the first config file up the tree is cuml's. `--rootdir=umap-upstream` properly sets the rootdir, but cuml's config file is still used.

FWIW I'm ok with the current solution, just flagged it to explain it during review.